README.md

```diff
@@ -594,6 +594,20 @@ second entry, the error structure should look like this:
 }
 ```
 
+Additionally, you can assign errors to a field using your own functions.
+
+```js
+this.formState = form.state(o, {
+    getError: accessor => accessor.raw === "wrong" ? "Is wrong" : undefined
+})
+```
+
+The `error` property of the field will contain the "Is wrong" error message if
+the field return `true` with the `getError` function. If a field contains both
```

Comment:
that's both not grammatical ("if the field return") and incorrect. The getError function (not the field) needs to return a string or undefined, not 'true'.